js/src/main/scala/crystal/react/implicits/package.scala

```diff
@@ -1,59 +1,59 @@
 package crystal.react
 
 import crystal._
+import crystal.react.reuse.Reuse
 import cats.MonadError
-import cats.syntax.all._
-import japgolly.scalajs.react._
-import japgolly.scalajs.react.vdom.html_<^._
-import japgolly.scalajs.react.component.Generic.MountedSimple
-
-import scala.util.control.NonFatal
 import cats.effect.Sync
 import cats.effect.Async
 import cats.effect.SyncIO
 import cats.effect.std.Dispatcher
+import cats.syntax.all._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.component.Generic.MountedSimple
+import japgolly.scalajs.react.util.DefaultEffects.{ Async => DefaultA, Sync => DefaultS }
 import monocle.Lens
 import org.typelevel.log4cats.Logger
-import crystal.react.reuse.Reuse
 
+import scala.util.control.NonFatal
 import scalajs.js
 
 package object implicits {
-  implicit class CallbackToOps[A](private val self: CallbackTo[A]) {
+  implicit class DefaultSToOps[A](private val self: DefaultS[A]) {
     @inline def to[F[_]: Sync]: F[A] = Sync[F].delay(self.runNow())
 
     @inline def toStream[F[_]: Sync]: fs2.Stream[F, A] =
       fs2.Stream.eval(self.to[F])
   }
 
   implicit class ModMountedSimpleFOps[S, P](
-    private val self: MountedSimple[CallbackTo, P, S]
+    private val self: MountedSimple[DefaultS, DefaultA, P, S]
```

Comment:
Is `Default[S|A]` something from `scalajs-react`

Reply:
Yes. Actually they are aliases to `japgolly.scalajs.react.util.DefaultEffects.{Sync, Async}`.

As long as we use `core` they are in turn just aliases to `CallbackTo` and `AsyncCallback`. Using the `DefaultEffects` starts paving the way towards generic effect in case we go down that path.